content/src/main/content/jcr_root/apps/acs-commons/components/content/sitewide-component-props-example/.content.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could we move the example impl into the docs or ACS Samples project (and link from docs)?

Apologies if I missed this being addressed already